components/TheFooter.vue

```diff
@@ -185,13 +185,13 @@ const menuKodadot: Menu[] = [
 
   {
     name: $i18n.t('artist ambassador'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/technical/kodadots-programs/artist-ambassador-program',
+    url: 'https://hello.kodadot.xyz/technical/kodadots-programs/artist-ambassador-program',
     external: true,
   },
 
   {
     name: $i18n.t('ambassador program'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/technical/kodadots-programs/ambassador-program',
+    url: 'https://hello.kodadot.xyz/technical/kodadots-programs/ambassador-program',
```

Comment:
https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/ambassador-program